src/Session.php

```diff
@@ -55,10 +55,12 @@ public function isStarted()
     }
 
     /**
-     * Starts session driver.
+     * Starts session driver if session is not started.
      *
      * Calling any action before visiting a page is an undefined behavior.
      * The only supported method calls on a fresh driver are
+     * - resizeWindow()
+     * - maximizeWindow()
```

Comment:
if we want to allow this, we must add new tests in the driver testsuite calling these methods before visiting a page, to ensure it actually works in all drivers